services/common/src/components/project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,472 @@
+    };
+  };
+
+  const handleTransformPayload = (payload: any) => {
+    let payloadValues: any = {};
+    const updatedAuthorizations = transformAuthorizations(payload);
+    const values = removeNullValuesRecursive(payload);
+    payloadValues = {
+      ...values,
+      ...updatedAuthorizations,
+    };
+    delete payloadValues.authorizationTypes;
+    return payloadValues;
+  };
+
+  /* Transforms project summary authorizations to match the
+   *  shape of project summary authorization form values.
+   */
+  const transformProjectSummaryAuthorizations = (input) => {
+    const output: any = {};
+    input.forEach((authorization) => {
+      const authType = authorization.project_summary_authorization_type;
+
+      if (!output[authType]) {
+        output[authType] = { types: [], AMENDMENT: [], NEW: [] };
+      }
+
+      authorization.project_summary_permit_type.forEach((permitType) => {
+        if (!output[authType].types.includes(permitType)) {
+          output[authType].types.push(permitType);
+        }
+        if (permitType === "AMENDMENT") {
+          if (!authorization.amendment_changes) {
+            authorization.amendment_changes = [];
+          }
+          output[authType].AMENDMENT.push(authorization);
+          if (!output[authType].NEW) {
+            output[authType].NEW = [];
+          }
+        } else if (permitType === "NEW") {
+          output[authType].NEW.push(authorization);
+          if (!output[authType].AMENDMENT) {
+            output[authType].AMENDMENT = [];
+          }
+        }
+      });
+    });
+
+    if (output.MINES_ACT_PERMIT) {
+      output.MINES_ACT_PERMIT = input.filter(
+        (auth) => auth.project_summary_authorization_type === "MINES_ACT_PERMIT"
+      );
+    }
+
+    return output;
+  };
+
+  const handleRetryAMSSubmissionClicked = async () => {
+    setIsLoaded(false);
+
+    try {
+      const transformedAuthorizations = transformProjectSummaryAuthorizations(
+        project.project_summary.authorizations
+      );
+      const projectSummary = {
+        ...project.project_summary,
+        authorizations: transformedAuthorizations,
+      };
+      const payload = handleTransformPayload({
+        ...projectSummary,
+        ams_terms_agreed: true,
+        confirmation_of_submission: true,
+      });
+
+      // Normalize contacts' addresses
+      payload.contacts.forEach((contact) => {
+        if (Array.isArray(contact.address)) {
+          contact.address = contact.address.length === 0 ? null : contact.address.join(", ");
+        }
+      });
+
+      // Normalize facility operator's address
+      if (Array.isArray(payload.facility_operator.address)) {
+        payload.facility_operator.address =
+          payload.facility_operator.address.length === 0
+            ? null
+            : payload.facility_operator.address[0];
+      }
+      await dispatch(
+        updateProjectSummary(
```

Comment:
I believe this endpoint also sends notifications - they should probably not be sent when retrying the jobs as we're not really making any other changes to the project summary

Reply:
the notification has been turned off now when a retry is attempted